base/applications/taskmgr/taskmgr.c

```diff
@@ -896,46 +880,45 @@ void TaskManager_OnRestoreMainWindow(void)
     SetWindowPos(hMainWnd, (OnTop ? HWND_TOPMOST : HWND_TOP), 0, 0, 0, 0, SWP_NOSIZE | SWP_NOMOVE | SWP_SHOWWINDOW);
 }
 
-void TaskManager_DisableStatusBar(HWND hWnd)
+void TaskManager_OnMenuSelect(HWND hWnd, UINT nItemID, UINT nFlags, HMENU hSysMenu)
 {
-    int nParts;
-
-    /* Update the status bar pane sizes */
-    nParts = -1;
-    SendMessageW(hStatusWnd, SB_SETPARTS, 1, (LPARAM) (LPINT)&nParts);
-    SendMessageW(hStatusWnd, SB_SETTEXT, (WPARAM)0, (LPARAM)L"");
-}
+    WCHAR str[100] = L"";
 
-void TaskManager_EnableStatusBar(HWND hWnd)
-{
-    RECT   rc;
-    int    nParts[3];
+    /*
+     * Reset the status bar if we close the current menu, or
+     * we open the system menu or hover above a menu separator.
+     * Adapted from comctl32!MenuHelp().
+     */
+    if ((LOWORD(nFlags) == 0xFFFF && hSysMenu == NULL) ||
+        (nFlags & (MF_SEPARATOR | MF_SYSMENU)))
+    {
+        /* Set the status bar for multiple-parts output */
+        SendMessageW(hStatusWnd, SB_SIMPLE, (WPARAM)FALSE, (LPARAM)0);
+        bTrackMenu = FALSE;
 
-    /* Update the status bar pane sizes */
-    GetClientRect(hWnd, &rc);
-    nParts[0] = STATUS_SIZE1;
-    nParts[1] = STATUS_SIZE2;
-    nParts[2] = rc.right;
-    SendMessageW(hStatusWnd, SB_SETPARTS, 3, (LPARAM) (LPINT) nParts);
+        /* Trigger update of status bar columns and performance page asynchronously */
+        RefreshPerformancePage();
+        return;
+    }
 
-    /* trigger update of status bar columns and performance page asynchronously */
-    RefreshPerformancePage();
-}
+    // if (nFlags & MF_POPUP) { ... }
```

Comment:
```suggestion
```
What is the purpose of that commented out line? Either remove it, or add a comment behind it what is going to be implemented here later, in case it would be a preparation of some thing you want to add here later.